packages/web/template.env

```diff
@@ -3,7 +3,7 @@ REACT_APP_URL="localhost:3007"
 REACT_APP_API_URL="/api"
 REACT_APP_SOCKET_URL="/api/socket.io"
 
-REACT_APP_SENTRY_DSN="DSN URL"
+REACT_APP_SENTRY_DSN="https://fa4694414d9249bf9c5534034f702ede@o1079031.ingest.sentry.io/6083593"
```

Comment:
Rebasing to the development should solve this issue. Aleks updated frontend so it doesn't crash without sentry dns.